### DISCUSSION TITLE
esp32 standard module

### DISCUSSION THREAD

**Discussion by parabyte** [pschatzmann/arduino-audio-tools#1691] [Q&A] [answered]
im playing around still with 8bit i2s.

I cannot Format Convert and use Helix at the same time audio is stuttering, even with a huge buffer.

#include "AudioCodecs/CodecMP3Helix.h"
AudioInfo to(11025, 1, 8); // Resampling to 8000 Hz, 8-bit, 2 channels
FormatConverterStream formatConverter(i2s);

is this due to lack of cpu cycles? or likely a bug in my code?

**Comment by pschatzmann** [comment, marked as answer]
I never used 8 bits, so I am not sure if this is a good idea. What do you expect to gain from this ?
Did you try to use a NumberFormatConverterStream instead ?
I also hope that you were setting the log level to Warning or Error...

ps. The ESP32 I2S api supports 8 bit, but there are many DACs that don't, so test this first with a Sine Generator with 8 and 16 bits, if you get the same tone!
